synapse/handlers/room_member.py

```diff
@@ -341,9 +341,10 @@ def _update_membership(
         prev_events_and_hashes = yield self.store.get_prev_events_for_room(
             room_id,
         )
-        latest_event_ids = (
+        latest_event_ids = [
```

Comment:
why is this changing?

Reply:
Ah, this used to be a required change but no longer. Will change it back for simplicity of PR